media-types/image/jxl.json

```diff
@@ -0,0 +1,43 @@
+{
+  "media-types": {
+    "image": {
+      "jxl": {
```

Comment:
FWIW @ddbeck I think this should be `jxl`, because the file extension is also `jxl`.